website/assets/styles/pages/docs/basic-documentation.less

```diff
@@ -340,7 +351,7 @@
         padding-bottom: 24px;
       }
 
-      h1 + ul {
```

Comment:
huh I never even knew `+` was supported in our targeted browsers, but apparently it's been supported for quite some time  TIL @gillespi314 @eashaw